### PR TITLE
Adding patch command to resolve nokogiri install issues in ruby image

### PR DESCRIPTION
**Pull request by ewolinetz**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -22,6 +22,7 @@ RUN rpmkeys --import file:///etc/pki/rpm-gpg/RPM-GPG-KEY-CentOS-7 && \
 	mysql-devel \
 	mysql-libs \
 	openssl-devel \
+        patch \
```
Comment (bparees) [inline Dockerfile]: fix the spacing please.

File: Dockerfile.rhel7 [hunk 2]
```diff
@@ -20,6 +20,7 @@ RUN yum install -y --setopt=tsflags=nodocs \
 	mysql-devel \
 	mysql-libs \
 	openssl-devel \
+        patch \
```
Comment (bparees) [inline Dockerfile.rhel7]: fix the spacing please.

**Comment by bparees** [comment]
[test]

**Comment by ewolinetz** [comment]
Fixed spacing

**Comment by bparees** [comment]
ok to test
add to whitelist

**Comment by bparees** [comment]
please squash your commits

**Pull request merged**